This file is part of MXE.
See index.html for further information.

Insist that ssize_t is defined.

diff --git a/util/cairo-missing/cairo-missing.h b/util/cairo-missing/cairo-missing.h
index 7e4f0a3..c20b869 100644
--- a/util/cairo-missing/cairo-missing.h
+++ b/util/cairo-missing/cairo-missing.h
@@ -37,6 +37,8 @@
 #include <string.h>
 #include <sys/types.h>
 
+#define _SSIZE_T_DEFINED 1
+
 #ifdef _WIN32
 #define WIN32_LEAN_AND_MEAN
 #include <windows.h>
